airbyte-integrations/connectors/source-adaptive-insights/source_adaptive_insights/schemas/export_data.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "account_name": {
+      "type": ["null", "string"]
+    },
+    "account_code": {
+      "type": ["null", "string"]
+    },
+    "level": {
+      "type": ["null", "string"]
+    },
+    "gl_account": {
```

Comment:
The extra dimensions of 
- Location
- Contract
- Assignment 
are missing on purpose? 

Or do you just want to test it with one at first?